releases/note-source/v18_0_0.rst

```diff
@@ -192,4 +205,23 @@ The ``--silent`` argument used to disable upload of metrics from :command:`ap_ve
 The capability to upload metrics has been removed from :command:`ap_verify.py` (see :jira:`DM-16536`), but ``--silent`` has been retained as a no-op for compatibility reasons.
 It will be removed before the next release.
 
+.. _release-v18-0-0-deprecate-isr:
+
+Upcoming removal of `ip_isr` functions from `isrFunctions.py`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These functions are replaced by functionality in `lsst.meas.algorithms.Defects`:
+
+- defectListFromFootprintList
```

Comment:
Can you link to the relevant docs for these functions? i.e., `~lsst.meas.algorithms.Defects.defectListFromFootprintList`, or similar?

Failing that, it would be nice to mark these as `code literals`.

Reply:
The docs to the replacement in meas_algorithms or the docs to the function in ip_isr that is disappearing?

Reply:
Oh, good point; I should read before commenting.

Obviously, it'd be crazy to link the name of one function to the docs for another. I guess it would be nice to have:

> `~lsst.ip.isr.functionFoo` is replaced by `~lsst.meas.algorithms.Defects.FunctionBar`

At least in so far as a one-to-one mapping is possible.

Reply:
for the ip_isr functions there are direct replacements in `Defects`. For the meas_algorithms one it's gone with no replacement.